ChangeLog

```diff
@@ -1,3 +1,75 @@
+
+- Features added since Pacemaker-2.0.4
+  + cibsecret: read value to set from input (or stdin) if not specified
+  + controller: add new IPC API command for getting node list
+  + libpe_status: Add type="integer" to rule elements, allowing for
+				  specifying 64-bit integers.
+  + scheduler: Add rule-based tests to op_defaults and rsc_defaults.
+  + scheduler: Add on-fail=demote and no-quorum-policy=demote recovery
+               policies for promoted resources.
+  + tools: Add --resource= to filter crm_mon output for a resource.
+  + tools: Add -P to crmadmin to ping via pacemakerd-api.
+  + tools: Use glib for cmdline handling in crmadmin and crm_resource.
+
+- Fixes for regressions introduced in Pacemaker-2.0.4
+  + scheduler: Add the node name back to bundle instances.
+  + tools: get stonith_admin --list-targets working again
+
+- Fixes for regressions introduced in Pacemaker-2.0.3
+  + scheduler: Fix adding HTTP headers to crm_mon in daemon mode.
+
+- Fixes for regressions introduced in Pacemaker-2.0.1
+  + scheduler: require pre-/post-start notifications correctly.
+
+- Changes since Pacemaker-2.0.4
+  + CIB: use crm_exit to free qb-logging
+  + cibsecret: don't use pssh -q option unless supported
+  + controller: Log correct timeout for timed-out stonith monitor
+  + controller: properly detect remote node info requests
+  + daemons: Accept PCMK_OCF_DEGRADED and PCMK_OCF_DEGRADED_MASTER status codes
+  + daemons: Notify tengine of completion of CRM_OP_LRM_DELETE.(CLBZ:#5424)
+  + daemons: Remove relayed stonith operation.(CLBZ#5401)
+  + executor: only send executor notifications to executor clients
+  + fencer:  Add an additional check to advance_topology_level.
+  + fencer: avoid infinite loop if device is removed during operation
+  + fencer: avoid possible use-of-NULL when parsing metadata
+  + ipc-api: allow calling connect after disconnection
+  + ipc-api: avoid infinite loop when disconnected
+  + libCIB: Add `tags` and `alerts` as known CIB paths, remove `domains`.
+  + libcrmcommon: Allow default formatted output message handlers.
+  + libcrmcommon: Don't write out name="" as part of XML output.
+  + libcrmcommon: Rewrite parse_op_key to understand migrate_to and migrate_from.
+  + libcrmcommon: use correct sense of string comparison in crm_action_str()
+  + libcrmcommon: Use truly implicit deny for ACLs
+  + libfencing: add `port` or `plug` parameter according to metadata on
+                `validate` if no `pcmk_host_argument` specified
+  + libfencing: respect `pcmk_host_argument=none` on `validate`
+  + libpacemaker: Don't allow a potential NULL in a format string.
+  + libpe_status: Don't output unnecessary resource headers.
+  + libpe_status: Handle filtering of inactive resources.
+  + libpe_status: Support integer as rule type attribute
+  + libpe_status: Use pcmk_monitor_timeout as stonith start timeout
+  + pacemakerd: improve logging when synced with SBD
+  + sbd-integration: sync pacemakerd with sbd
```

Comment:
This should be under features as "support SBD_SYNC_RESOURCE_STARTUP environment variable to better synchronize Pacemaker start-up and shutdown with SBD"